core/src/main/scala/org/bitcoins/core/hd/BIP32Path.scala

```diff
@@ -144,6 +147,21 @@ object BIP32Path {
 
     BIP32PathImpl(path)
   }
+
+  def fromBytes(bytes: ByteVector): BIP32Path = {
+    require(bytes.size % 4 == 0, "ByteVector is not suited for KeyPath")
```

Comment:
it's always good to include the thing we are requiring an invariant on in the error message, like this:

```scala
require(bytes.size % 4 == 0, s"ByteVector is not suited for KeyPath, got=${bytes.length}")
```